src/Components/Article_card/Article_card.jsx

```diff
@@ -1,5 +1,63 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import { useParams, Link } from "react-router-dom";
+import { getArticles, getArticlesById } from "../../utils/API";
+import Article_solo from "../Article_solo/Article_solo";
+import {
+  Wrapper,
+  Content,
+  Title,
+  Button,
+  AuthorTopic,
+  Votes,
+} from "./Article_card.styles";
 
-export default function Article_card() {
-  return <div>Article_card</div>;
-}
+const Article_card = () => {
+  const [state, setState] = useState([]);
```

Comment:
I would personally rename this to be articles, setArticles. 

state, setState doesn't tell me what sort of state or data you are trying to set here.